source/2019-09-20-ember-3-13-released.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Ember 3.13 Released
+author: Kenneth Larsen
+tags: Releases, 2019, 3, 3.13
+responsive: true
+---
+
+Today the Ember project is releasing version 3.13 of Ember.js, Ember Data, and Ember CLI. 
+
+Additionally, versions 3.12 of Ember and Ember Data are now promoted to LTS, which stands for Long Term Support. An LTS version of Ember continues to receive security updates for 9 release cycles (54 weeks) and bugfixes for 6 cycles (36 weeks). LTS releases typically occur every four minor versions. The previous LTS versions for Ember and Ember Data were 3.4.
+
+If you are upgrading from the previous LTS version to 3.12 then you are getting access to these features:
```

Comment:
When I first read this, I thought it was saying 3.6, 3.8, 3.10, 3.11 are all LTS versions, which confused me.

Reply:
(I think the list is good/interesting, but the way it is worded/presented was a bit confusing to me)